### PR TITLE
Enhancements and refactor for Angular Universal Support

### LINKED ISSUE #12810: Angular Universal Support

### PR DESCRIPTION
**Pull request by cetincakiroglu**
Fixes #12810

### CODE REVIEW

**Comment by volvachev** [comment]
Hi mb add [package](https://github.com/ng-web-apis/common) to use tokens like `WINDOW`.

File: src/app/showcase/doc/dock/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/dock/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/editor/propsdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/editor/propsdoc.ts:1]: Revert change

File: src/app/showcase/doc/blockui/documentdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/blockui/documentdoc.ts:1]: Revert change

File: src/app/showcase/doc/breadcrumb/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/breadcrumb/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/contextmenu/documentdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/contextmenu/documentdoc.ts:1]: Revert change

File: src/app/showcase/doc/contextmenu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/contextmenu/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/megamenu/megamenuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/megamenu/megamenuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/menu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/menu/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/menubar/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/menubar/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/panelmenu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/panelmenu/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/slidemenu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/slidemenu/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/steps/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/steps/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/table/exportdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/table/exportdoc.ts:1]: Revert changes

File: src/app/showcase/doc/tabmenu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/tabmenu/menuitemdoc.ts:1]: Revert change

File: src/app/showcase/doc/theming/scalingdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/theming/scalingdoc.ts:1]: Revert change

File: src/app/showcase/doc/tieredmenu/menuitemdoc.ts
Comment (kyjus25) [inline src/app/showcase/doc/tieredmenu/menuitemdoc.ts:1]: Revert change

**Review by kyjus25 — CHANGES_REQUESTED** [review]
Several files here have a copy-paste change to documentation files that should remain the word "document".

**Comment by cetincakiroglu** [comment]
> 

Thanks for indicating that copy-paste issues. All fixed

**Pull request merged**